docs/modules/operation/pages/quick-start/visualize-data.adoc

```diff
@@ -31,49 +31,45 @@ The rest of the dashboard updates dynamically.
-** *Duration:* how long the dashlet displays in the rotation, in seconds.
-** *Boost-Priority:* change the dashlet's priority if it is in the alert state.
-This setting is optional, and is not available for all dashlets.
-** *Boost-Duration:* change the display duration if the dashlet is in the alert state.
-This setting is optional, and is not available for all dashlets.
-** (Optional) Click *Properties* to configure additional settings (alarm severity, chart type, and so on).
-. Click the *Up Arrow* and *Down Arrow* symbols to change the dashlet's order.
-This affects its rotation order in the Ops Board view, or its position in the Ops Panel view.
-. Click *Preview* in the dashlet settings area to preview the dashlet.
-. Click *Preview* beside the operator board name to preview the board and all of its dashlets.
-
-The board's configuration is automatically saved.
-To view the ops board, click menu:Dashboards[Ops Board] in the menu bar.
-
-For more information about ops boards and dashlets, see xref:operation:deep-dive/admin/webui/opsboard/introduction.adoc[Operator Board] in the Deep Dive section.
+For more information on each of the components, see xref:deep-dive/admin/webui/dashboard.adoc[Dashboard] in the Deep Dive section.
 
 == Surveillance View
 
-The Surveillance View is similar to its component of the same name in the <<#qs-visualize-dashboard, Dashboard>>.
+The Surveillance View is similar to its component of the same name in the <<#qs-visualize-dashboard, dashboard>>.
 Click menu:Status[Surveillance] in the top menu bar to view it.
 
 Just like in the Dashboard, you can select rows, columns, and cells to narrow your data focus.
 
-For more information, see xref:operation:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.
+For more information, see xref:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.
+
+=== Visualize data using surveillance categories
+
+The Surveillance View and its corresponding dashboard component visualize all routers, switches, and servers in your production, test, and development environments by default (see <<deep-dive/admin/webui/surveillance-view.adoc#surveillance-view-default-config, Default configuration>>).
+You can create custom surveillance categories to define specific data that you want to visualize in the Surveillance View.
+Follow these steps to create a new surveillance category:
+
+. Click the *gear* symbol in the top-right of the screen.
+. Under Provisioning, click *Manage Surveillance Categories*.
+. Enter a name for the new category in the *Category Name* box, and click *Add New Category*.
+. Add nodes to the category as desired.
+** Choose nodes from the *Available Nodes* list and click *Add*.
+
+Your changes are automatically saved.
+
+Once you have created a custom surveillance category, you can create a new Surveillance View to update the data that is visualized:
```

Comment:
Could this also include a specific example? Probably so they can select the category that they created in the previous section.